kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -155,6 +155,17 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
     SessionHandle(HandleIdentifier(UUID.fromString(batchId), STATIC_BATCH_SECRET_UUID), protocol)
   }
 
+  def getBatchSessionList(batchType: String, from: Int, size: Int): List[Session] = {
+    allSessions().filter { session =>
```

Comment:
the ordering is not stable, we should do sort by batch id or other key

Reply:
Thanks, sort with batch id.